### ISSUE TITLE
Using Quizlet, the keyboard disappears/hides after Enter

### ISSUE THREAD

**Issue by Cyberman-tM** [AnySoftKeyboard/AnySoftKeyboard#1020]
### Steps to reproduce
1. Install Quizlet
2. write something and press enter
3. keyboard disappears until you tap the input line again

### Expected behaviour
The keyboard should stay

### Actual behaviour
The keyboard disappears (hides downward)

**Android OS version:**
4.1.2  (old device)
6.0.1 (new device)

**Device manufacturer and model:**
LG P880 (old device)
LG K500Y

**List of installed add-ons (like languages, or themes):**
none
(I had the klingon keyboard installed, but removed during tests)

I *think* that maybe somehow Quizlet issues something like a "hardware button pressed" event? Or "I'm done, close keyboard"? It isn't a recent issue, I think it's been that way at least half a year.

I'm not entirely certain if it only happens with ASK, or also with the regular keyboard - I *think* it happens when ASK is installed, even if it's not the active keyboard - somehow I'm not really getting consistent results..

**Comment by menny** [comment]
I'm not sure what I should try.
I'm not familiar with this app. I installed it, but I don't see anything I can tap to verify the bug.

**Comment by Cyberman-tM** [comment]
Right, I forgot that not everyone knows or has every app I use - the obvious stuff is so easily forgotten :-(

You need to "subscribe" to a lesson or set:
Use the search button to look for one - for example the Spanish vocabulary that is suggested (to me?).
I get a list of sets, just tap the first one, then you should get a new screen with "Learn", "Flashcards" and so on.
Tap "write".
Now what I wrote above applies.

**Comment by Cyberman-tM** [comment]
I noticed something that might help - the input field quizlet uses doesn't seem to be a text field.

When switching to the standard keyboard (LG Keyboard) the "enter" button changes to "Go" during Quizlet input . I've seen this "Go" button in the browser URL for example.

So maybe the behaviour of ASK is correct and it's a mis-used type of input field by Quizlet?

**Comment by AnXh3L0** [comment]
Does this happen anymore with the latest versions of ASK and Quizlet @Cyberman-tM?
